tests/typecheck/lostno/LostNo.java

```diff
@@ -1,16 +1,13 @@
-package typecheck.lostno;
+package universe.lostno;
 
 import universe.qual.*;
 
 public class LostNo {
-    //:: error: (uts.explicit.lost.forbidden)
-    @Lost Object lo;
```

Comment:
Add it again

Reply:
-Alint=allowLost perhaps doesn't forbid explicit annotation + add a comment

Reply:
Please also add a comment with the option and file an issue.